apps/nextra/pages/en/build/cli/setup-cli/install-move-prover.mdx

```diff
@@ -3,6 +3,20 @@ import { Callout } from 'nextra/components'
 # Install the Move Prover
 
 If you want to use the [Move Prover](../../smart-contracts/prover.mdx), install the Move Prover dependencies after [installing the CLI binary](.).
+There are two ways to install Prover dependencies.
+
+## Installation through Aptos CLI (Recommended)
+
+1. Install the latest Aptos CLI binary.
```

Comment:
Might be nice to link this text to a page with instructions to install the latest Aptos CLI binary.

Reply:
done